src/Testing/src/LoggedTest/LoggedTestBase.cs

```diff
@@ -95,6 +95,12 @@ public virtual void Initialize(TestContext context, MethodInfo methodInfo, objec
                 _initializationException = ExceptionDispatchInfo.Capture(e);
             }
         }
+        
+        public virtual Task InitializeAsync(TestContext context, MethodInfo methodInfo, object[] testMethodArguments, ITestOutputHelper testOutputHelper)
+        {
+            Initialize(context, methodInfo, testMethodArguments, testOutputHelper);
+            return Task.CompletedTask;
+        }
 
```

Comment:
```suggestion
        public virtual Task InitializeAsync(TestContext context, MethodInfo methodInfo, object[] testMethodArguments, ITestOutputHelper testOutputHelper)
        {
            Initialize(context, methodInfo, testMethodArguments, testOutputHelper);
            return InitializeCoreAsync(context);
        }
        
        protected virtual Task InitializeCoreAsync() => Task.CompletedTask;
```
We do this in other places, the advantage is that you don't have to remember to call base.